src/Client/PullPayment.php

```diff
@@ -0,0 +1,272 @@
+<?php
+
+declare(strict_types=1);
+
+namespace BTCPayServer\Client;
+
```

Comment:
import `PreciseNumber` needed in createPullPayment below
`use BTCPayServer\Util\PreciseNumber;`